lib/data_update_scripts/20201209134953_touch_articles_with_capitalized_paths.rb

```diff
@@ -0,0 +1,13 @@
+module DataUpdateScripts
+  class TouchArticlesWithCapitalizedPaths
+    def run
+      offending_articles = Article.where("path != lower(path)")
+
+      # Some log visibility for troubleshooting/monitoring
+      Rails.logger.info("Updating #{offending_articles.count} articles...")
```

Comment:
```suggestion
      Rails.logger.info("Updating #{offending_articles.count} articles with capitalized paths...")
```
**non-blocking suggestion:** this might make it a tiny bit more useful when parsing the logs for this in case something _does_ go wrong